### PR TITLE
PR for US1

### PR DESCRIPTION
**Pull request by Leviron**

### CODE REVIEW

File: components/Card/Card.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+"use client";
```
Comment (tejue) [inline components/Card/Card.js:1]: I am not familiar with "use client" so I hope it works as you planned :)

File: components/Card/Card.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <>
+      {recipes.map((recipe) => (
+        <ContainerStyle key={recipe.id}>
+          <Image
+            src={`/images/${recipe.picture}`}
+            width={60}
+            height={50}
+            alt={recipe.title}
+          />
+          <div className="ContainerStyle--text">
```
Comment (tejue) [inline components/Card/Card.js:19]: In our sessions we learned to implement styling (when working with styled-components like this:
`<ContainerStyle 
...rest of the code
/>` 

meaning without className but I am not sure how this works with ContainerStyle--text

File: components/lib/recipes.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      "1 tsp dried basil",
+      "Salt and pepper",
+      "Olive oil",
+    ],
+    instructions:
+      "1. Finely chop the onion and garlic, then sauté them in olive oil. Add the ground beef and cook until browned. \n2. Stir in the tomato paste, canned tomatoes, oregano, basil, salt, and pepper. Simmer for 20-30 minutes. \n3. Meanwhile, cook the spaghetti according to the package instructions. \n4. Serve the Bolognese sauce over the drained spaghetti and enjoy.",
+  },
+  {
+    id: 2,
+    picture: "caesar-salad.png",
```
Comment (tejue) [inline components/lib/recipes.js:36]: You might want to use the same order as in id: 1 even though it is not relevant for an object. But maybe it is helpful for yourself

File: components/lib/recipes.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+      "1 tsp Dijon mustard",
+      "1 garlic clove",
+      "Salt and pepper",
+    ],
+    instructions:
+      "1. Wash the lettuce leaves, dry them, and tear them into bite-sized pieces. \n2. Cut the garlic clove in half and rub the salad bowl with it. \n3. In the bowl, whisk together olive oil, lemon juice, Worcestershire sauce, mustard, salt, and pepper. \n4. Add the lettuce to the bowl and toss everything together. Top with croutons and grated Parmesan cheese before serving.",
+  },
+  {
+    id: 3,
+    picture: "curry.png",
```
Comment (tejue) [inline components/lib/recipes.js:55]: same here, see above :)

**Review by tejue — APPROVED** [review]
Hey Ron, I am getting hungry when seeing the pictures :spaghetti:
Well done.

I am not familiar or just not sure with some lines of your code so I just left you some comments.

File: components/lib/recipes.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
+class RecipeClass {
```
Comment (ekat-br) [inline components/lib/recipes.js:1]: Warum arbeitest du mit Klassen? Wir haben das im Kurs nie behandelt. Ich würde dir davon abraten.

**Pull request merged**